File: resourcespoet/src/test/java/com/commit451/resourcespoet/RemoveTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

import java.io.File;

/**
 * Tests the resources creation
 */
public class RemoveTest {

    @Test
    public void removeTest() throws Exception {
        ClassLoader classLoader = getClass().getClassLoader();
        File file = new File(classLoader.getResource("remove_before.xml").getFile());
        ResourcesPoet poet = ResourcesPoet.create(file)
                .remove(Type.STRING, "red");

        TestUtil.assertEquals("remove_after.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/RemoveTest.kt
package com.commit451.resourcespoet

import org.junit.Ignore
import org.junit.Test

import java.io.File

/**
 * Tests the resources creation
 */
class RemoveTest {

    @Test
    @Ignore("broken due to indentations. Fix it!")
    fun removeTest() {
        val classLoader = javaClass.classLoader
        val file = File(classLoader.getResource("remove_before.xml")!!.file)
        val poet = ResourcesPoet.create(file)
            .remove(Type.STRING, "red")

        TestUtil.assertEquals("remove_after.xml", poet)
    }
}
